Combined_NN_Model.py

```diff
@@ -1,70 +1,32 @@
 ### import the required modules and functions
-#TODO: Is this the correct Write_Performance()? --> NO, it's not. We should
-# update it with the parameters important for the NN in Utils.py
-# Same for NN_param_typecheck in NN_Utils.py
+
 import subprocess
 import time
 import sys
-
-from pycorenlp import StanfordCoreNLP
-
 from Utils import Write_Performance
 from config import *
-from reddit_parser import Parser
 from ModelEstimation import NNModel
 from transformers import BertTokenizer
 from NN_Utils import *
+from reddit_parser import Parser # Does the parser object need to be adjusted?
```

Comment:
The comment at the end of this line should be deleted

Reply:
Done